packages/client/guestCheckout_test.ts

```diff
@@ -240,29 +241,6 @@ describe({
           },
         },
       });
-      const stream = guestRelayClient.createEventStream();
```

Comment:
the checkout test was a false positive test because of this extra stream declaration